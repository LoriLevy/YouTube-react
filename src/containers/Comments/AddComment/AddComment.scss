@import '../../../styles/shared.scss';

.add-comment {
    display: flex;
    margin: 16px auto;

    form {
        //flex: 1;  /*  This means take up the whole remaining free space  */
        width: 82vw;
    }

    .user-image {
        width: $avatar-diameter;
        height: $avatar-diameter;
        margin-right: $avatar-margin;
        margin-left: 15px;
    }
}