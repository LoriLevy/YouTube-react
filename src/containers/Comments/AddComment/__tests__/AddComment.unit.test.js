import React from 'react';
import {shallow} from 'enzyme';
import {AddComment} from '../AddComment';

describe('AddComment', () => {
    test('It renders correctly', () => {
        const wrapper = shallow(<AddComment/>);
        expect(wrapper).toMatchSnapshot();   
    });
});