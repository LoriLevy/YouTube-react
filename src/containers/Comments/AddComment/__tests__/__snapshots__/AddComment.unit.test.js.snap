// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddComment It renders correctly 1`] = `
<div
  className="add-comment"
>
  <Image
    as="img"
    circular={true}
    className="user-image"
    src="http://via.placeholder.com/48x48"
    ui={true}
  />
  <Form
    as="form"
  >
    <FormTextArea
      as={[Function]}
      autoHeight={true}
      control={[Function]}
      placeholder="Add your comment"
    />
  </Form>
</div>
`;
